test/proxy_configuration.test.js

```
@@ -206,9 +241,9 @@ describe('Apify.createProxyConfiguration()', () => {
     test('should throw group is not available', async () => {
```

should throw when group is not available

---

fixed.